client/app/scripts/superdesk-search/search.js

```diff
@@ -544,34 +561,41 @@
                                 });
                             }
 
-                            _.forEach(scope.items._aggregations.priority.buckets, function(priority) {
-                                scope.aggregations.priority[priority.key] = priority.doc_count;
-                            });
+                            if (angular.isDefined(scope.items._aggregations.priority)) {
+                                _.forEach(scope.items._aggregations.priority.buckets, function(priority) {
+                                    scope.aggregations.priority[priority.key] = priority.doc_count;
+                                });
+                            }
 
-                            _.forEach(scope.items._aggregations.source.buckets, function(source) {
-                                scope.aggregations.source[source.key] = source.doc_count;
-                            });
-                            if (angular.isDefined(scope.items._aggregations.source))
-                            {
+                            if (angular.isDefined(scope.items._aggregations.priority)) {
```

Comment:
should this be `angular.isDefined(scope.items._aggregations.source` instead of `angular.isDefined(scope.items._aggregations.priority`?